### PR TITLE
apimtemplate: Add version 0.1-alpha

### PR DESCRIPTION
**Pull request by MichielVanwelsenaere**

### CODE REVIEW

File: bucket/apimtemplate.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "description": "Azure API Management DevOps Resource Kit",
+    "license": "MIT",
+    "version": "0.1-alpha",
+    "url": "https://github.com/Azure/azure-api-management-devops-resource-kit/releases/download/v0.1-alpha/azure-api-management-devops-example-win-x64-v0.1-alpha.zip",
+    "hash": "8b980e59eec011c5eed042e8dc576b76e22b871f8dccefec6e4181dfcc281889",
+    "depends": "azure-cli",
+    "bin": [
+        "apimtemplate.exe"
+    ],
+    "checkver": "github",
```
Comment (niheaven) [inline bucket/apimtemplate.json]:
```suggestion
    "checkver": {
        "url": "https://github.com/Azure/azure-api-management-devops-resource-kit/releases",
        "regex": "tag/(?:v)?([\\w.-]+)"
    },
```

It is prerelease, `github` mode is not suitable.

File: bucket/apimtemplate.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "autoupdate": {
+        "url": "https://github.com/Azure/azure-api-management-devops-resource-kit/releases/download/v$version/azure-api-management-devops-example-win-x64-v$version.zip"
```
Comment (niheaven) [inline bucket/apimtemplate.json]:
```suggestion
        "url": "https://github.com/Azure/azure-api-management-devops-resource-kit/releases/download/v$version/azure-api-management-devops-example-win-x64-v$version.zip",
        "hash": {
            "url": "$url.sha2"
        }
```

hash should be get from hash file.

File: bucket/apimtemplate.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    },
+    "notes": [
```
Comment (niheaven) [inline bucket/apimtemplate.json]: Don't add useage notes in manifest, just add installation notes.

File: bucket/apimtemplate.json (hunk 1 above)
Comment (MichielVanwelsenaere) [reply bucket/apimtemplate.json]: done

File: bucket/apimtemplate.json (hunk 2 above)
Comment (MichielVanwelsenaere) [reply bucket/apimtemplate.json]: done

**Pull request merged**